go.mod

```diff
@@ -82,7 +82,7 @@ require (
 	k8s.io/cli-runtime v0.26.0
 	k8s.io/client-go v0.26.0
 	k8s.io/cloud-provider v0.25.2
-	k8s.io/code-generator v0.25.0
+	k8s.io/code-generator v0.25.9
```

Comment:
It is expected to have mixed version here?

```
    k8s.io/api v0.26.0
	k8s.io/apiextensions-apiserver v0.26.0
	k8s.io/apimachinery v0.26.0
	k8s.io/cli-runtime v0.26.0
	k8s.io/client-go v0.26.0 # <----- v0.26.0
	k8s.io/cloud-provider v0.25.2 # <----- v0.25.2
	k8s.io/code-generator v0.25.9 # <----- v0.25.9
```

fyi, 0.25.11 is out
- https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md

Reply:
> It is expected to have mixed version here?

The versions won't be mixed, the actual ones that are being used sit under `replace ( ... )` section.

> fyi, 0.25.11 is out

Thanks for pointing this out, updated the PR accordingly.